.rubocop.yml

```diff
@@ -0,0 +1,17 @@
+AllCops:
+  TargetRubyVersion: 2.3
```

Comment:
we supports ruby from 2.2 to 2.4 .. 2.3 style syntax works without a warning for all of those